From ace6bb4b6fa0ff09d38bd31c26c21dd81aaadca6 Mon Sep 17 00:00:00 2001
From: Wolf <wolf@wolfsden.cz>
Date: Wed, 20 Jul 2022 02:00:52 +0200
Subject: [PATCH 1/4] Do not use prebuilt binaries

Prebuilt binaries (ijar, singlejar) for linux are glibc specific,
meaning they do not work on alpine. Change the select to use _cc_binary
version instead.
---
 tools/jdk/BUILD.tools | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/jdk/BUILD.tools b/tools/jdk/BUILD.tools
index ad3e61e41a..7586fc22de 100644
--- a/tools/jdk/BUILD.tools
+++ b/tools/jdk/BUILD.tools
@@ -168,7 +168,6 @@ alias(
 alias(
     name = "ijar_prebuilt_binary_or_cc_binary",
     actual = select({
-        "//src/conditions:linux_x86_64": ":ijar_prebuilt_binary_linux",
         "//src/conditions:darwin": ":ijar_prebuilt_binary_darwin",
         "//src/conditions:windows": ":ijar_prebuilt_binary_windows",
         "//conditions:default": "@remote_java_tools//:ijar_cc_binary",
@@ -198,7 +197,6 @@ alias(
 alias(
     name = "singlejar_prebuilt_or_cc_binary",
     actual = select({
-        "//src/conditions:linux_x86_64": ":prebuilt_singlejar_linux",
         "//src/conditions:darwin": ":prebuilt_singlejar_darwin",
         "//src/conditions:windows": ":prebuilt_singlejar_windows",
         "//conditions:default": "@remote_java_tools//:singlejar_cc_bin",
-- 
2.36.2

